GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -137,6 +135,40 @@ - (nullable NSString *)openIDRealm {
   return [_authState.lastTokenResponse.request.additionalParameters[kOpenIDRealmParameter] copy];
 }
 
+- (NSString *)idTokenString {
+  return _authState.lastTokenResponse.idToken;
+}
+
+- (nullable NSDate *)idTokenExpirationDate {
```

Comment:
Personally, I don't think you need `Date` at the end of this. 'Expiration` already connotes time.

Reply:
As an example, consider `+[NSDate distantFuture]`. The word 'Future' already tells you to that the thing returned is related to time. The getter's signature tells you it is an `NSDate`. Under the naming you're using here, this method would look like, `+[NSDate distantFutureDate]`, which reads redundantly to me.